### PR TITLE
feat: footer redesigned 😎  #235

### LINKED ISSUE #235: feat: redesign footer section
We can give our project a good Footer which will be not much fancy it will be attractive and responsive @priyankarpal

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
I have redesigned the Footer Section of the page. Added Discord and GitHub icon and also MIT license

fixed #235

### CODE REVIEW

**Comment by Bharadwajshivam28** [comment]
okay

**Comment by Bharadwajshivam28** [comment]
like i have to remove built for collaboration line ????

**Comment by priyankarpal** [comment]
> like i have to remove built for collaboration line ????

changed text to `Built for collaboration 🤝🏼`

**Comment by Bharadwajshivam28** [comment]
okay

**Comment by Bharadwajshivam28** [comment]
Screenshot from 2023-04-24 23-50-44

is it okay???

**Comment by Bharadwajshivam28** [comment]
I am facing issue in aligning github icon to end/right most with discord icon??

**Comment by priyankarpal** [comment]
> Screenshot from 2023-04-24 23-50-44
> 
> is it okay???

Then you don't need to add icons. I will add this.

**Comment by priyankarpal** [comment]
I fixed

**Comment by Bharadwajshivam28** [comment]
I didnt got your point

**Comment by priyankarpal** [comment]
> I didnt got your point

I will add the icons. & I changed the footer texts

**Comment by Bharadwajshivam28** [comment]
okay so should i remove both discord and github icon

**Comment by Bharadwajshivam28** [comment]
> > okay so should i remove both discord and github icon
> 
> yes

okay then i am removing both icons and updating it

**Comment by priyankarpal** [comment]
> > > okay so should i remove both discord and github icon
> > 
> > 
> > yes
> 
> okay then i am removing both icons and updating it

yeah do it.

**Comment by Bharadwajshivam28** [comment]
> > > > okay so should i remove both discord and github icon
> > > 
> > > 
> > > yes
> > 
> > 
> > okay then i am removing both icons and updating it
> 
> yeah do it.

ok

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
-        </a>
-        <span> &copy; {new Date().getFullYear()} - Open Source</span>
-      </p>
+    <footer aria-label="Site Footer" className="bg-white dark:bg-gray-900">
+      <div className="max-w-screen-xl px-4 pt-16 pb-8 mx-auto sm:px-6 lg:px-8 lg:pt-24">
+        <div className="text-center">
+          <h2 className="text-3xl font-extrabold text-white sm:text-5xl">proudly open source</h2>
+
+          <p className="max-w-sm mx-auto mt-4 text-gray-400">
+            ProjectsHut is an open source project. Feel free to contribute to the project.
```
Comment (Deva-1903) [inline src/components/Footer.jsx]:
```suggestion
            ProjectsHut is an open-source project. Feel free to contribute and help us make this project even better.
```

**Review by Deva-1903 — CHANGES_REQUESTED** [review]

**Comment by Bharadwajshivam28** [comment]
thanks for pointing @Deva-1903

**Comment by Bharadwajshivam28** [comment]
Screenshot from 2023-04-25 00-31-15

@priyankarpal is it alright?

**Comment by Bharadwajshivam28** [comment]
ya sure making it to center

**Comment by Bharadwajshivam28** [comment]
MIT license should be in center correct??

**Comment by priyankarpal** [comment]
> MIT license should be in center correct??

yes whole text

**Comment by priyankarpal** [comment]
> Screenshot from 2023-04-25 00-36-39
> 
> Is it alright??

yes.

**Comment by Bharadwajshivam28** [comment]
ok creating a push and updating it

**Comment by Deva-1903** [comment]
> > Screenshot from 2023-04-25 00-31-15
> > @priyankarpal is it alright?
> 
> if you can make the text centre do it. & change the text that @Deva-1903 reviewed

Did you 

> > Screenshot from 2023-04-25 00-31-15
> > @priyankarpal is it alright?
> 
> if you can make the text centre do it. & change the text that @Deva-1903 reviewed

Change the text and push @Bharadwajshivam28

**Comment by Bharadwajshivam28** [comment]
I will push now

**Comment by priyankarpal** [comment]
@Bharadwajshivam28 Whenever you create an issue/PR try to add a clear title & description

**Comment by Bharadwajshivam28** [comment]
okay @priyankarpal

**Comment by Bharadwajshivam28** [comment]
@priyankarpal creating a fresh pr

**Comment by priyankarpal** [comment]
> @priyankarpal creating a fresh pr

ok then close this

**Pull request closed without merging**